### PR TITLE
add tracking param data

### PR DESCRIPTION
**Pull request by jdorweiler**

### CODE REVIEW

File: docs/DATA_MODEL.md [hunk 1]
```diff
@@ -171,6 +171,23 @@ Generated files created while building Tracker Radar
...
 |[api_fingerprint_scores](/build-data/generated/api_fingerprint_scores.json)|An object mapping browser APIs to their likelihood to be used for fingerprinting. Higher score means that an API is more likely to be used for fingerprinting|
+|[tracking-parameters](/build-data/generated/tracking_parameters.json)|An object listing how prevalent a parameter is used in third party requests and cookies.|
+
+### Tracking parameters
+
+Tracking parameters are found by generating parameter values, and adding these to the crawl site URLs. When crawl data is processed these generated values can be found in cookies and third party requests.
+
+- **totalSites** - total number of sites processed
+- **params** -  an object listing request and cookie tracking for each parameter. Keys in the params object are the parameter name.
+  - prevalence - percent of totalSites that this parameter was seen in cookies or third party requests
```
Comment (kdzwinel) [inline docs/DATA_MODEL.md:182]: both first and third party cookies are taken into account here?

File: docs/DATA_MODEL.md [hunk 2]
```diff
@@ -171,6 +171,23 @@ Generated files created while building Tracker Radar
...
+  - exampleSites - list of example sites where either requests or cookies were found
+  - requests3p
+      - prevalence - percent of totalSites where this parameter was seen in a third party request
+      - entities - top 10 entities using this parameter in requests
+  - cookies
+      - prevalence - percent of totalSites where this parameter was seen in a cookie
+      - firstParty - count of first party cookies with this parameter
+      - thirdParty - count of third party cookies with this parameter
```
Comment (kdzwinel) [inline docs/DATA_MODEL.md:190]: there are also entities on cookies - does it make sense to list 1p entities?

File: build-data/generated/tracking_parameters.json [hunk 3]
```diff
@@ -0,0 +1,902 @@
...
+                "www.merriam-webster.com",
+                "puzzles.msn.com",
+                "vimeo.com",
+                "community.babycenter.com/post/a7884385/straining_crying_during_bowel_movements_for_breastfed_newborn",
+                "www.travelmath.com/time-change/from/Spain/to/Argentina"
+            ],
+            "requests3p": {
+                "prevalence": 0.285,
+                "entities": {
+                    "Google LLC": 0.743,
```
Comment (kdzwinel) [inline build-data/generated/tracking_parameters.json:21]: I start to wonder if domains wouldn't be better here - e.g. I think it'd be nice to know if it's google-analytics or google ads or GTM or firebase collecting those IDs I think?

File: build-data/generated/tracking_parameters.json [hunk 4]
```diff
@@ -0,0 +1,902 @@
...
+                    "The Nasdaq Stock Market LLC": 0.05,
+                    "Auth0, Inc.": 0.05,
+                    "urban-vpn.com": 0.05,
+                    "expresso.pt": 0.05,
+                    "lawofselfdefense.com": 0.05
+                }
+            }
+        },
+        "fb_source": {
+            "prevalence": 0.096,
```
Comment (kdzwinel) [inline build-data/generated/tracking_parameters.json:590]: it *almost* seemsl like you sorted params by prevalence, but suddenly this? Is that expected?

File: build-data/generated/tracking_parameters.json [hunk 5]
```diff
@@ -0,0 +1,902 @@
...
+                    "WarnerMedia, LLC": 0.017,
+                    "Amazon Technologies, Inc.": 0.015
+                }
+            },
+            "cookies": {
+                "prevalence": 0.001,
+                "firstParty": 33,
+                "thirdParty": 15,
+                "entities": {
+                    "concealedcarry.com": 0.267,
```
Comment (kdzwinel) [inline build-data/generated/tracking_parameters.json:802]: the number here is no longer the same prevalence as above, right? This 0.267 is percentage of that 0.001?

**Review by kdzwinel — COMMENTED** [review]
Mmmm… this data is tasty! 🍤 I left some comments.

File: docs/DATA_MODEL.md (hunk 1 above)
Comment (jdorweiler) [reply docs/DATA_MODEL.md:182]: Yes

File: docs/DATA_MODEL.md [hunk 6]
```diff
@@ -171,6 +171,23 @@ Generated files created while building Tracker Radar
...
+|[tracking-parameters](/build-data/generated/tracking_parameters.json)|An object listing how prevalent a parameter is used in third party requests and cookies.|
+
+### Tracking parameters
+
+Tracking parameters are found by generating parameter values, and adding these to the crawl site URLs. When crawl data is processed these generated values can be found in cookies and third party requests.
+
+- **totalSites** - total number of sites processed
+- **params** -  an object listing request and cookie tracking for each parameter. Keys in the params object are the parameter name.
+  - prevalence - percent of totalSites that this parameter was seen in cookies or third party requests
+  - exampleSites - list of example sites where either requests or cookies were found
```
[the remark this answers is not in the document]
Comment (jdorweiler) [reply docs/DATA_MODEL.md:183]: Before cutting the list down, yes. The final list of example sites is cut down to just 10.

File: docs/DATA_MODEL.md (hunk 2 above)
Comment (jdorweiler) [reply docs/DATA_MODEL.md:190]: I didn't include them. It's kinda the same thing as the list of example sites.

File: build-data/generated/tracking_parameters.json (hunk 4 above)
Comment (jdorweiler) [reply build-data/generated/tracking_parameters.json:590]: It's not sorted

File: build-data/generated/tracking_parameters.json (hunk 5 above)
Comment (jdorweiler) [reply build-data/generated/tracking_parameters.json:802]: > This 0.267 is percentage of that 0.001?
correct

File: build-data/generated/tracking_parameters.json (hunk 3 above)
Comment (jdorweiler) [reply build-data/generated/tracking_parameters.json:21]: We can do both

Comment (kdzwinel) [reply build-data/generated/tracking_parameters.json:21]: this is great, thank you!

File: build-data/generated/tracking_parameters.json [hunk 7]
```diff
@@ -0,0 +1,1135 @@
...
+                    "US Department of State": 0.009,
+                    "ViacomCBS Inc.": 0.007,
+                    "USDA": 0.006,
+                    "Harvard University": 0.005,
+                    "Wondershare Technology Co., Ltd.": 0.004,
+                    "ARNOLDO MONDADORI EDITORE SpA": 0.004
+                },
+                "domains": {
+                    "www.realtytrac.com": 0.202,
+                    "www.www.realtytrac.com": 0.101,
```
Comment (kdzwinel) [inline build-data/generated/tracking_parameters.json:60]: that looks a bit off - `www.www.realtytrac.com` doesn't load. Maybe this is a bug in trackers code, but I think it's worth double checking

File: build-data/generated/tracking_parameters.json [hunk 8]
```diff
@@ -0,0 +1,1135 @@
...
+                "account.mail.ru",
+                "www.rottentomatoes.com/tv/the_wheel_of_time",
+                "stacks.cdc.gov",
+                "pay.amazon.com",
+                "news.google.de"
+            ],
+            "requests3p": {
+                "prevalence": 0.095,
+                "entities": {
+                    "Google LLC": 0.205,
```
Comment (kdzwinel) [inline build-data/generated/tracking_parameters.json:336]: kinda weird that we see facebook collecting `fbclid` but not any of those other `fb_*` params - just an observation

File: docs/DATA_MODEL.md [hunk 9]
```diff
@@ -171,6 +171,26 @@ Generated files created while building Tracker Radar
...
+  - requests3p
+      - prevalence - percent of totalSites where this parameter was seen in a third party request
+      - entities - top 10 entities using this parameter in requests
+      - domains - top 10 domains using this parameter in requests
+  - cookies
+      - prevalence - percent of totalSites where this parameter was seen in a cookie
+      - firstParty - count of first party cookies with this parameter
+      - thirdParty - count of third party cookies with this parameter
+      - entities - top 10 entities using this parameter in cookies
+      - domains - top 10 domainss using this parameter in cookies
```
Comment (kdzwinel) [inline docs/DATA_MODEL.md:193]:
```suggestion
      - domains - top 10 domains using this parameter in cookies

```

**Review by kdzwinel — APPROVED** [review]
Thanks for adding domains to the data! I left couple more comments, all nits, feel free to merge after that 🤘

**Pull request merged**